ct/omada.sh

```diff
@@ -29,19 +29,14 @@ function update_script() {
   fi
 
   msg_info "Updating MongoDB"
-  MONGODB_VERSION="7.0"
-  if ! lscpu | grep -q 'avx'; then
-    MONGODB_VERSION="4.4"
-    msg_error "No AVX detected: TP-Link Canceled Support for Old MongoDB for Debian 12\n https://www.tp-link.com/baltic/support/faq/4160/"
-    exit 1
+  if lscpu | grep -q 'avx'; then
+    MONGODB_VERSION="8.0"
+    msg_ok "AVX detected: Using MongoDB 8.0"
+    MONGO_VERSION="8.0" setup_mongodb
+  else
+    MONGO_VERSION="4.4" setup_mongodb
   fi
```

Comment:
why not log only if no avx detected -->

```suggestion
  if lscpu | grep -q 'avx'; then
    MONGODB_VERSION="8.0"
    MONGO_VERSION="8.0" setup_mongodb
  else
    msg_warn "No AVX detected: Using old MongoDB 4.4"
    MONGO_VERSION="4.4" setup_mongodb
fi
```

as far as I understand this correctly?

Reply:
When avx -> use 8, when not Fallback to 4.4?